feat(changelog): instances-added-openssh-server-is-now-enabled-on-window 2025-02-04 (#4350)
scaleway/docs-content: changelog/february2025/2025-02-04-instances-added-openssh-server-is-now-enabled-on-window.mdx

---

> title: OpenSSH server is now enabled on Windows 2022  images

Reviewer: ```suggestion
title: OpenSSH server now preinstalled on Windows 2022 images
```

status: added
author:

>     fullname: ''

Reviewer: ```suggestion
    fullname: 'Join the #instances channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2025-02-04
category: compute
product: instances
---

> The newest release of the Windows 2022 server and Windows 2022 core images now have the OpenSSH Server and OpenSSH Client packages installed by default.
> 
> The OpenSSH Server is installed and disabled by default for security purposes. It can be enabled at instance creation time by adding the tag `with-ssh` to the instance when it is created.
> 
> This is an example of a CLI command to start a server with OpenSSH Server enabled :
> ```
> scw instance server create name=win2k22-core image=windows-server-2022-core tags.0=with-ssh type=POP2-2C-8G-WIN admin-password-encryption-ssh-key-id={ssh_key_id}
> ```
> If your instance has been started without the tag and you want to enable OpenSSH Server, simply use the following Powershell command :
> 
> ```
> Start-Service sshd
> ```
> You will then need to load the ssh keys configured in your project by using the ScalewayEcosystem command :
> ```
> PS C:\Users\administrator> Scw-Fetch-SSH-Keys

Reviewer: ```suggestion
The latest release of **Windows Server 2022** and **Windows Server 2022 Core** images now includes the **OpenSSH Server** and **OpenSSH Client** packages by default.  

- The **OpenSSH Server** is installed but **disabled by default** for security reasons.  
- To enable it at instance creation, add the tag `with-ssh` when launching your server.  

#### Enabling OpenSSH at Instance creation  

Use the following CLI command to create a new Instance with OpenSSH Server enabled:  

```sh
scw instance server create name=win2k22-core image=windows-server-2022-core tags.0=with-ssh type=POP2-2C-8G-WIN admin-password-encryption-ssh-key-id={ssh_key_id}
```

#### Enabling OpenSSH on an existing Instance  

If your instance was started **without** the `with-ssh` tag, you can enable OpenSSH manually by running the following PowerShell command:  

```powershell
Start-Service sshd
```

Once started, load your SSH keys configured in your project using the following command:  

```powershell
PS C:\Users\administrator> Scw-Fetch-SSH-Keys
```

This ensures your SSH keys are loaded and ready for authentication.  
```
